modules/benchmark/R/metric.R2.R

```diff
@@ -1,10 +1,12 @@
 ##' @name metric.R2
-##' @title Coefficient of Determination
+##' @title Coefficient of Determination (R2)
 ##' @export
 ##' @param dat dataframe
 ##' 
 ##' @author Betsy Cowdery
+
 metric.R2 <- function(dat, ...) {
+  logger.info("Metric: Coefficient of Determination (R2)")
```

Comment:
@bcow just a nit picky thing.  Since we can't use superscript, what if it was instead (R_squared) to be descriptive and avoid any possible confusion....not a necessary change just thought I would voice my opinion